src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="row">
+  <div class="col-sm-12 col-lg-12">
+      <div>
+          <cd-table-actions class="btn-group mb-4"
+              [permission]="permission"
+              [selection]="selection"
+              [tableActions]="createTableActions">
+          </cd-table-actions>
+      </div>
+    <div class="card">
+      <div class="card-header"
+           i18n>Multisite Topology viewer</div>
+      <div class="card-body">
+        <div class="row">
+          <div class="col-sm-6 col-lg-6 tree-container">
+            <i *ngIf="loadingIndicator"
+               [ngClass]="[icons.large, icons.spinner, icons.spin]"></i>
+
+            <tree-root #tree
+                       [nodes]="nodes"
+                       [options]="treeOptions"
+                       (updateData)="onUpdateData()">
+              <ng-template #treeNodeTemplate
+                         let-node>
+                  <span *ngIf="node.data.name">
+                      {{ node.data.name }}&nbsp;&nbsp;
```

Comment:
its better again to stick with bootstrap margins rather than `&nbsp` for consistency. I am okay with this but this will look even more better if you add a margin at right.
```suggestion
                  <span *ngIf="node.data.name"
                         class="me-1">
                      {{ node.data.name }}
```